src/pages/term/Term.tsx

```diff
@@ -14,19 +19,23 @@ type TermProps = {
  * The term page template of the website.
  */
 export default function Term(props: TermProps) {
+  //TODO: implement these dummy methods once the representation of terms is final.
+  const visitNextItem = () => {};
+
+  const visitPreviousItem = () => {};
+
   const [term, setTerm] = useState<ITerm>();
   useChangeTitle(term?.displayName);
   useChangeDescription(term?.description);
 
-  const [markdown, setMarkdown] = useState<string>();
+  const [markdown, setMarkdown] = useState<string | null>(null);
```

Comment:
`markdown` never is assigned to `null`, it either `string` or `undefined`.
```suggestion
  const [markdown, setMarkdown] = useState<string>();
```